docs/docsite/rst/network_debug_troubleshooting.rst

```diff
@@ -495,7 +495,7 @@ If the user requires a password to go into privileged mode, this can be specifie
 
 Add `authorize: yes` to the task. For example:
 
-.. code-block:: yaml
+.. code-block:: yaml+jinja
```

Comment:
Note: yaml+jinja2 should be hte default value (highlight_language defined in conf.py)

Reply:
It wants a pygments lexer name, valid options are ('yaml+jinja' or 'YAML+Jinja' or 'salt') http://pygments.org/docs/lexers/#pygments.lexers.templates.YamlJinjaLexer

conf.py currently uses highlight_language 'YAML+Jinja'

Not sure how 'sticky' the code-block lang is, but this is explicit if nothing else.

Reply:
code-block lang is not sticky.  There's a separate way to change the highlight language within a sphinx doc and then that is sticky for the rest of the document.